### PR TITLE
Delete staging directory if it already exists

### LINKED ISSUE #11: Clear out the .docker_staging directory if it already exists
Currently, running the build pipeline on a directory that already contains .docker_staging will cause the build to fail.

### PR DESCRIPTION
**Pull request by aslo**
fixes #11

### CODE REVIEW

File: java-runtime-builder-app/src/main/java/com/google/cloud/runtimes/builder/buildsteps/docker/StageDockerArtifactBuildStep.java [hunk 1]
```diff
@@ -61,8 +63,11 @@ protected void doBuild(Path directory, Map<String, String> metadata) throws Buil
       logger.info("Found artifact {}", artifact);
 
       // make staging dir
-      // TODO delete dir if exists
-      Path stagingDir = Files.createDirectory(directory.resolve(DOCKER_STAGING_DIR));
+      Path stagingDir = directory.resolve(DOCKER_STAGING_DIR);
+      if (Files.exists(stagingDir)) {
+        deleteDirectoryRecursive(stagingDir);
```
Comment (meltsufin) [inline java-runtime-builder-app/src/main/java/com/google/cloud/runtimes/builder/buildsteps/docker/StageDockerArtifactBuildStep.java]: We should put in a log statement here telling the user about the delete.

File: java-runtime-builder-app/src/main/java/com/google/cloud/runtimes/builder/buildsteps/docker/StageDockerArtifactBuildStep.java [hunk 2]
```diff
@@ -90,6 +95,23 @@ protected void doBuild(Path directory, Map<String, String> metadata) throws Buil
   }
 
   /*
+   * Deletes a directory and all of its contents.
+   */
+  private void deleteDirectoryRecursive(Path path) throws IOException {
```
Comment (meltsufin) [inline java-runtime-builder-app/src/main/java/com/google/cloud/runtimes/builder/buildsteps/docker/StageDockerArtifactBuildStep.java]: Can you use Apache Commons IO `FileUtils.deleteDirectory` instead?

Comment (aslo) [inline java-runtime-builder-app/src/main/java/com/google/cloud/runtimes/builder/buildsteps/docker/StageDockerArtifactBuildStep.java]: I could, but I did just remove the Apache Commons IO dependency. Do you think it's worth re-adding that dependency for just this one method?

File: java-runtime-builder-app/src/main/java/com/google/cloud/runtimes/builder/buildsteps/docker/StageDockerArtifactBuildStep.java (hunk 1 above)
Comment (aslo) [inline java-runtime-builder-app/src/main/java/com/google/cloud/runtimes/builder/buildsteps/docker/StageDockerArtifactBuildStep.java]: :+1:

File: java-runtime-builder-app/src/main/java/com/google/cloud/runtimes/builder/buildsteps/docker/StageDockerArtifactBuildStep.java (hunk 2 above)
Comment (meltsufin) [inline java-runtime-builder-app/src/main/java/com/google/cloud/runtimes/builder/buildsteps/docker/StageDockerArtifactBuildStep.java]: It's up to you. I don't mind re-adding it. If you decide to keep this method, you should write a test for it that exercises it directly. Also, the method should be public static and moved to a Utils class or something.

**Pull request merged**